lib/core/test/data/Cardano/Wallet/Api/ApiNetworkClock.json

```diff
@@ -0,0 +1,71 @@
+{
+    "seed": 6322276116191378239,
+    "samples": [
+        {
+            "ntp_status": {
+                "syncing": "completed",
```

Comment:
What about:

- calling the top-level field `ntp`
- using `status` as a tag

Such that we have:

```json
{
    "status": "pending"
}

{
    "status": "unavailable"
}

{
    "status": "available",
    "offset": {
      "quantity": 14,
      "unit": "microseconds"
    }
}
```

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/1396/commits/572a9be7b99c90baf1b96e51b94f0d82b1cfd635